documents-webapp/src/main/webapp/vue-app/documents/components/body/table/cells/DocumentsFileNameCell.vue

```diff
@@ -254,10 +254,7 @@ export default {
       if (this.file && this.file.folder){
         this.$root.$emit('document-open-folder', this.file);
       } else {
-        let id = this.file.id;
-        if (this.file.sourceID){
-          id = this.file.sourceID;
-        }
+        const id = this.file.id;
```

Comment:
check this [commit](https://github.com/exoplatform/documents/commit/6ba66ad99760d79cd6062bd1fc6108e493def269#diff-bbbe756cfeab1ec5fcb9eda9377fe51993bce6cce0d0a810072129c77d505c69L264), we should verify if that this modification will not cause a regression

Reply:
duplicating a shared document works perfectly